CHANGELOG.md

```diff
@@ -21,71 +21,6 @@
-- DESTDIR and prefix support in the Makefile  ([2049e62](2049e626eb4719de582b6232a25572a7eb959707))
-- Missing default declarations of `k8s_ctr_address` and `k8s_namespace`  ([bd069fe](bd069fe6d4cf0bb44d7bda7315fa1f48945b7dad))
-
-
-
-
-### Documentation
-
-
-- *(compose)* Add documentation to compose configuration
-- Basic readme content  ([5647a62](5647a62affb7d63e13016daadadc36631cee1860))
-
-
-
-
-### Features
-
-
-- App update module generator generator. ([MEN-6080](https://northerntech.atlassian.net/browse/MEN-6080)) ([31c843c](31c843c04a74912cb9d0d335098cd2f06dedf156))
-- App Update Module. ([MEN-6076](https://northerntech.atlassian.net/browse/MEN-6076)) ([4d4c36e](4d4c36ee03aafa19fc8c56d5ad0416c8f18238d8))
-- Deep delta support: binary delta between image layers. ([MEN-6081](https://northerntech.atlassian.net/browse/MEN-6081)[MEN-6195](https://northerntech.atlassian.net/browse/MEN-6195)) ([f23d554](f23d5545189d9b320224bdd12bf29263ff46edf1))
-  We can significantly reduce the binary delta size by taking
-  into account corresponding layers inside the image.
-  
-  * add support for the generation of delta between layer.tar files
-  * modify LOAD operation in the docker-compose orchestrator sub-module
-- Bash test framework ([MEN-6082](https://northerntech.atlassian.net/browse/MEN-6082)) ([69951a4](69951a4359333d84f73bbe54650b21337e2c6c58))
-- Detecting composition errors and handling ([MEN-6367](https://northerntech.atlassian.net/browse/MEN-6367)) ([3f2a6b6](3f2a6b643cfee01adba355f4b0f4bd20bcfaadbc))
-- Configuration for orchestrator sub modules ([MEN-6366](https://northerntech.atlassian.net/browse/MEN-6366)) ([7ed8099](7ed809977f8ae0ef7dd5186b36bdcf6717370479))
-- Requirements check. ([MEN-6368](https://northerntech.atlassian.net/browse/MEN-6368)) ([add40cd](add40cd29cd8f78e8b45ee5e67e43fdace9241c3))
-- K8s support ([MEN-6194](https://northerntech.atlassian.net/browse/MEN-6194)) ([e1bb1eb](e1bb1ebb27dab628fdf6cb32ac461dd413db55df))
-- Cleanup, save, rollback. ([MEN-6077](https://northerntech.atlassian.net/browse/MEN-6077)) ([4538e3c](4538e3c8472d8a9ffd19ac42a3430136b555fb5a))
-
-
-
-
-### Testing
-
-
-- App-gen smoke test ([MEN-6079](https://northerntech.atlassian.net/browse/MEN-6079)) ([c812e1f](c812e1f2832de0be5bec2d58c6ee9530711cab64))
-- Update module ArtifactInstall test. ([MEN-6079](https://northerntech.atlassian.net/browse/MEN-6079)) ([3a3b944](3a3b94410b8a1a30e3bb2c195bd89ba33fe57532))
-- Fixing the expected shasums mechanism  ([b6d41da](b6d41daa0baf0eba36af3a5161e9fe5447aaa359))
-- Acceptance test. ([MEN-6082](https://northerntech.atlassian.net/browse/MEN-6082)) ([2055a31](2055a3136c8c73f6823875db7f3e08ab81bd8463))
-- Verify that no pulling appeared ([MEN-6556](https://northerntech.atlassian.net/browse/MEN-6556)) ([45ba921](45ba9214fd12a8fee74f9cf1750176bd3aadc384))
-
-
-
-
-### Refac
-
-
-- *(app)* Refactor assert_requirements to only check status code
-
-
-
+* First release of app-update-module
```

Comment:
This has to follow [this comment](https://github.com/mendersoftware/mender-docs-changelog/pull/78#discussion_r1751674910)